Source/Bogus.Tests/DataSetTests/DateTest.cs

```diff
@@ -1,16 +1,22 @@
 using System;
 using System.Globalization;
+using System.Linq;
 using Bogus.DataSets;
 using FluentAssertions;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Bogus.Tests.DataSetTests
 {
    public class DateTest : SeededTest
```

Comment:
Just an idea: We could also add a PInvokation to [SetTimeZoneInformation](https://learn.microsoft.com/en-us/windows/win32/api/timezoneapi/nf-timezoneapi-settimezoneinformation) (on Windows) to change the runner's time zone which would allow for covering all time zones e.g. in a  data-driven manner.
This affects system settings though, so I am not sure how compatible this is with the CI runners.

Reply:
I am horrified by the idea of unit tests altering system-wide settings. :-P We are between a rock and a hard place on this one. In my opinion, only enabling the tests when the ambient environment supports them is the lesser evil.

Reply:
In my opinion, the correct solution to this is to have the test-running environment prepare virtual machines with desired configuration on which to run the tests. This, of course, doesn't solve the problem for running tests locally, but _that_ I feel _is_ a place where if a developer really needs it, they can temporarily (manually) change their time zone settings.